src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -229,12 +229,12 @@ TER Transactor::apply ()
     }
 
     auto const& fees = view().fees();
-    mFeeDue = STAmount (getApp().getFeeTrack().scaleFeeLoad(
-        calculateBaseFee(), fees.base, fees.units, view().flags() & tapADMIN));
+    mFeeDue = getApp().getFeeTrack().scaleFeeLoad(
+        calculateBaseFee(), fees.base, fees.units, view().flags() & tapADMIN);
```

Comment:
Isn't this supposed to be done elsewhere? Before it even gets into the open ledger?